pages/docs/installation/installation-packages.md

```diff
@@ -42,7 +42,21 @@ These packages are maintained by the preCICE community and may be occasionally o
 However, we appreciate the effort and you may be able to contribute to them.
 
 - [MSYS2](https://packages.msys2.org/base/mingw-w64-precice) (for Windows, built with MinGW), [thread on our forum](https://precice.discourse.group/t/precice-and-mingw-packages/382)
-- [Nix / NixOS](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&query=precice)
 - [EasyBuild](https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/p/preCICE)
 - [Conda](https://anaconda.org/conda-forge/precice) (see also packages [pyprecice](https://anaconda.org/conda-forge/pyprecice) and [fenicsprecice](https://anaconda.org/conda-forge/fenicsprecice))
 - [FreeBSD](https://www.freshports.org/science/precice)
+
+### Nix / NixOS
+
+In addition to the community efforts listed above, preCICE and several of the bindings and adapters are available for Nix in the [precice/nix-packages repository](https://github.com/precice/nix-packages/).
+
+If you are using Nix (or NixOS for that matter), you can simply run the command `nix shell github:precice/nix-packages#precice-aster github:precice/nix-packages#precice-openfoam-adapter` for instance, which will drop you into a shell with code-aster and the OpenFOAM adapter accessible.
+All available outputs of the Nix flake can be shown by running `nix flake show github:precice/nix-packages`.
+
+The Nix flake also contains the outputs `vm`, `iso` and `vagrant-vbox-image`, which output a VM in several formats containing a remake of the [preCICE distribution v2211.0](installation-vm.html).
+To run the preCICE VM you can, on a NixOS system, simply run `nix run github:precice/nix-packages`.
+Note that this will build all the components needed for the preCICE VM locally on your machine, if they cannot be fetched from the upstream binary cache.
+A full build of the VM can take several hours, depending on the hardware used.
+For more options have a look at the [readme](https://github.com/precice/nix-packages#readme) in the repository.
+
+For all packages available upstream, see the [NixOS search](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&query=precice).
```

Comment:
Are the packages from `precice/nix-packages` going to be (automatically) published on nixos.org? If yes, then it would make more sense to bring first this link, and later describe the repository.

Reply:
No, the packages are not published in any automated way. We decided not to submit them on purpose, so we could quickly make changes on our own without having to wait for upstream to adopt the changes.

Maybe the packages could be submitted upstream in the future, but this is not part of the scope.